debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouter.java

```diff
@@ -127,13 +128,23 @@ public R apply(R r) {
             }
         }));
 
+        boolean isDeleteEvent = payload == "" || payload == null;
```

Comment:
Why should we check for empty string?

Reply:
So the check for empty string makes it so that the payload column can have a not null constraint (which may be the case). I added it more as a safety for that scenario.

Reply:
Uh, oh, just realizing this uses "==" for string comparison. I'll fix it.